dist/debian/apparmor-userns

```diff
@@ -1,8 +1,8 @@
 # Permit unprivileged user namespace creation for apptainer starter
-abi <abi/4.0>,
+abi <abi/3.0>,
```

Comment:
It's my understanding that only abi/4.0 supports the "userns" entry, since abi/3.0 works even without it.  So leave this as abi/4.0.

Alternatively we could consider using only the "placeholder" contents everywhere, but I'm not sure if that might have side effects.

Reply:
let's keep the "userns" entry in case of any defunct cases.